backend/src/test/java/com/hjjang/backend/email/EmailRegexTest.java

```diff
@@ -0,0 +1,62 @@
+package com.hjjang.backend.email;
+
+import static org.assertj.core.api.Assertions.*;
+
+import com.hjjang.backend.domain.email.exception.MailException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class EmailRegexTest {
+
+	private static final String REGEX = "^[a-zA-Z0-9]+@[a-zA-Z0-9]+\\.+a+c+\\.+k+r+$";
+	private static final String EMAIL_PARSE_REGEX = "[@.]";
+
+	@Test
+	void parseUniversity() {
+		//given
+		String email = "32174294@dankook.ac.kr";
+		//when
+		String university = List.of(email.split(EMAIL_PARSE_REGEX)).get(1);
+		//then
+		System.out.println("university = " + university);
+		assertThat(university).isEqualTo("dankook");
+	}
+
+	@Test
+	void checkInvalidEmail() {
+		//given
+		//when
+		List<String> falseEmails = new ArrayList<>();
+		falseEmails.add("32174294@naver.com");
+		falseEmails.add("32174294@gmail.com");
+		falseEmails.add("32174294_dankook.ac.kr");
+		falseEmails.add("@dankook.ac.kr");
+		falseEmails.add("32174294@ac.kr");
+		falseEmails.add("32174294@.ac.kr");
```

Comment:
```suggestion
		falseEmails.add("32174294@.ac.kr");
		falseEmails.add("dr0joon@gs.anyang.ac.kr");
```

제 대학교 이메일인데 요즘 저런 구글의 Gsuit 메일을 쓰는 경우가 생기는것 같더라고요
이 경우도 테스트 케이스에 포함되면 좋을거 같아서요